docs/troubleshooting.md

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting
+
+## Deploying into an existing VPC
+
+The initialization step of `kubeadm`, `kubeadm init` requires your node to be
```

Comment:
this might not be a true statement...but something is timing out and this is my best guess with the error message i've seen.

Reply:
Do you have the kubelet logs for the failure? It could be a mismatch between the node name and the attribute used for identifying the node with the cloud provider. I know this was an issue a while back and not sure if has been resolved in more recent releases.

Reply:
It appears that it is still attempting to determine the node name from the ec2 api, specificaly using the PrivateDnsName attribute of the instance.